_rules/meta-viewport-b4f0c3.md

```diff
@@ -24,20 +24,26 @@ acknowledgments:
 
 This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
 
-- the `content` [attribute value][] has the `user-scalable` property; or
-- the `content` [attribute value][] has the `maximum-scale` property.
+- the `content` attribute specified value includes the text `user-scalable`; or
+- the `content` attribute specified value includes the text `maximum-scale`.
+
+**Note:** The Applicability refers to string inclusion of the indicated strings in the value of the `content` attribute, as written.
 
 ## Expectation 1
 
 For each test target, the [attribute value][] does not have a `user-scalable` property with a value of `no`.
 
+The [attribute value][] of the `content` attribute is a list of key-value pairs, obtained by parsing the specified value, then applying the [translations into `@viewport` descriptors][descriptor translation], notably [the translation for `user-scalable`](https://www.w3.org/TR/css-device-adapt-1/#user-scalable).
+
 ## Expectation 2
 
 For each test target, the [attribute value][] does not have a `maximum-scale` property with a value less than 2.
 
+The [attribute value][] of the `content` attribute is a list of key-value pairs, obtained by parsing the specified value, then applying the [translations into `@viewport` descriptors][descriptor translation], notably [the translation for `maximum-scale`](https://www.w3.org/TR/css-device-adapt-1/#min-scale-max-scale) (including dropping the key-value pair completely for invalid values).
```

Comment:
Same issue here. What we want if the `max-zoom` description to not have a value less than 2. So, we can either say that in the expectation, or say that the `maximum-scale` property, must not have a numeric value under 2, or a literal value that is different from `device-width` and `device-height`

Reply:
Still, if we have an example like 

```
<meta name="viewport" content="maximum-scale=device-width">
```

how do I compare `device-width` with `2`?

Reply:
After translating it and using the value for `max-zoom`, `device-width` results in a `max-zoom` of 10, which is greater than 2, so this is OK 🤔

Reply:
I think only now it clicked in my head. I was using the definition of `attribute value` to compute the value. But the second paragraph adds to the definition. Should we, instead, update the definition of `attribute value` to compute the value in this instance that refers to the value of a CSS property instead of the value of the HTML attribute?

Reply:
I'm not sure because this is currently the only rule doing that, so I'd prefer to keep locality.
OTOH, we can definitely  improve the situation with some examples. I'll give it a go.